features/upgrade/api_auth/upgrade.feature

```diff
@@ -0,0 +1,52 @@
+Feature: apiserver and auth related upgrade check
+  # @author kewang@redhat.com
+  @upgrade-prepare
+  @admin
+  Scenario: Default RBAC role, rolebinding, clusterrole and clusterrolebinding without any missing after upgraded - prepare
+    When I run the :get admin command with:
+      | resource | clusterroles.rbac |
+      | o        | yaml              |
+    Then the output is parsed as YAML
+    And the output should contain:
+      | rbac.authorization.kubernetes.io/autoupdate: "true" |
+    And the output should not contain:
+      | rbac.authorization.kubernetes.io/autoupdate: "false" |
+    Given as admin I successfully patch resource "clusterrole.rbac/system:build-strategy-docker" with:
+      | {"rules": [{"apiGroups": ["","build.openshift.io"],"resources": ["builds/docker","builds/optimizeddocker"],"verbs": [ "get" ]}] } |
+    When I run the :get admin command with:
+      | resource      | clusterroles.rbac            |
+      | resource_name | system:build-strategy-docker |
+      | o             | yaml                         |
+    Then the output is parsed as YAML
+    And the expression should be true> @result[:parsed]['rules'][0]['verbs'][0] == "get"
+    Given as admin I successfully patch resource "clusterrolebinding.rbac/system:oauth-token-deleters" with:
+      | {"subjects":[{"apiGroup": "rbac.authorization.k8s.io","kind": "Group","name": "system:authenticated"}]} |
+    When I run the :get admin command with:
+      | resource      | clusterrolebinding.rbac     |
+      | resource_name | system:oauth-token-deleters |
+      | o             | yaml                        |
+    Then the output is parsed as YAML
+    And the expression should be true> @result[:parsed]['subjects'][0]['name'] == "system\:authenticated"
```

Comment:
A minor issue: the case main checkpoint is, if the default items are missing, they will be recovered after upgrade. Therefore what need be more ensured is: the default items are missing before upgraded. But your two `the expression should be true` lines only checked what items existed, didn't check what items are missing. I think this check needs be more preferred to be added.

Reply:
Will add more checking points here.

Reply:
Added the checking points for missed items.